defs/messages.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+message User {
+	string id = 1;
```

Comment:
Can we please use spaces please? :D

Reply:
Also, can we have an alias for generating all the definitions?

Reply:
What kind of alias?
Also, the space was being defined from `.proto` extension by my vim, as Go >.<

Reply:
The elixir files are generated with `protoc`, right? So we can have an alias to that `protoc` call with .protoc and target paths. I may be totally missing something :D

Reply:
Yeah, I can create a alias later. I will have to add the `mix format` after this one because they add an empty line after the `end`, so the CI wouldn't catch the `check formatting` step :P

Reply:
Yeah, it can be something like `proto: ["protoc ...", "format lib/livebook/web_socket"]` :)

Btw I wasn't suggesting running protoc on CI, but now that you mentioned it, we could also add a check to make sure the generated code is up to date, but we can do it later :)

Reply:
I meant to add the alias you already suggested :P

But, for the CI, I was thinking about checking based on `git diff` (but yeah, we can do it later)

Reply:
Bump, still need the alias :D